### ISSUE TITLE
Class1.cs -> UtilityLibraries.cs

### ISSUE THREAD

**Issue by Andrey-Doroshenko** [dotnet/docs.ru-ru#297]
Исправьте Class1.cs на UtilityLibraries.cs, а то не складывается результат.

---
#### Сведения о документе

⚠ *Не вносите правки в этот раздел. Это необходимо для связывания страницы сайта docs.microsoft.com с вопросом на GitHub.*

* ID: 8df4bc4b-88e1-a629-fcbd-6808b4f7308c
* Version Independent ID: 87d3afdd-d7d5-35d7-0718-63dadc4eab1d
* Content: [Создание библиотеки классов .NET Standard с помощью Visual Studio - .NET Core](https://docs.microsoft.com/ru-ru/dotnet/core/tutorials/library-with-visual-studio)
* Content Source: [docs/core/tutorials/library-with-visual-studio.md](https://github.com/dotnet/docs.ru-ru/blob/live/docs/core/tutorials/library-with-visual-studio.md)
* Product: **dotnet-core**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

**Comment by tdykstra** [comment]
Renaming the file is optional -- the library works as intended with the file name Class1.cs. Did you get an error message when you ran the app without renaming the file?

**Comment by Andrey-Doroshenko** [comment]
Yes, that's right. Debugging starts without error only after renaming.

**Comment by tdykstra** [comment]
The completed project is here -- https://github.com/dotnet/docs/tree/master/docs/core/tutorials/snippets/library-with-visual-studio/csharp
It runs fine for me in Visual Studio 2019 when I set ShowCase as the startup project.  What error message do you get, and what version of Visual Studio are you using?
